packages/app/src/LogSidePanel.tsx

```diff
@@ -1978,6 +1978,27 @@ function SidePanelHeader({
     parsedProperties?.['process.tag.rum.sessionId'] ??
     sessionId;
 
+  const headerEventTags = useMemo(() => {
+    return [
+      ['service', logData._service],
+      ['host', logData._host],
+      ['k8s.node', parsedProperties['k8s.node.name']],
+      ['k8s.pod', parsedProperties['k8s.pod.name']],
+      ['k8s.statefulset', parsedProperties['k8s.statefulset.name']],
+      ['k8s.container', parsedProperties['k8s.container.name']],
```

Comment:
maybe we want to keep '.name' so its consistent with other tags ?

Reply:
updated it (sorry for overwriting commit, i did it on autopilot 🤦 )